src/app/patient-dashboard/common/forms/form-order-metadata.service.ts

```diff
@@ -20,7 +20,7 @@ export class FormOrderMetaDataService {
 
         return this.formsOrder;
     }
-    public setFavouriteForm(name: string) {
+    public setFavouriteForm(name: any) {
```

Comment:
why switch to 'any'?

Reply:
it's still a string but build breaks when declared as a string but passes when it's of type any